File: FlaskDemoHanlp/HanlpHelper.py
#!/usr/bin/env python
# -*- coding:utf-8 -*-

"""
hanlp helper
"""

from HanlpDict import relation_dict, nature_dict


def relation_to_chinese(relation):
    if relation in relation_dict:
        return relation_dict.get(relation)
    else:
        print("relation translate not found ; "+relation)
        return relation


def tag_to_chinese(tag):
    if tag in nature_dict:
        return nature_dict.get(tag)
    else:
        print("tag translate not found ; "+tag)
        return tag

File: SpringBootDemoHanlp/src/main/java/com/neohope/nlp/refn/HanlpHelper.java
package com.neohope.nlp.refn;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class HanlpHelper {
	private static final Logger logger = LoggerFactory.getLogger(HanlpHelper.class);  

	public String relationToChinese(String relation){
	    if(HanlpDict.relationDict.containsKey(relation)){
	    	return HanlpDict.relationDict.get(relation);
	    }
	    else{
	        logger.warn("relation translate not found ; "+relation);
	        return relation;
	    }
	}

	public String tagToChinese(String tag){
		if(HanlpDict.natureDict.containsKey(tag)){
	        return HanlpDict.natureDict.get(tag);
		}
	    else{
	        logger.warn("tag translate not found ; "+tag);
	        return tag;
	    }
	}
	
}
    
